l1-contracts/contracts/governance/ServerNotifier.sol

```diff
@@ -27,26 +47,50 @@ contract ServerNotifier is Ownable2Step, ReentrancyGuard, Initializable {
         }
```

Comment:
I know it is not part of your PR, but could you please remove the `disableInitializers` thing